### DISCUSSION TITLE
Invariant failed: Draggable[id: 2]: Unable to find drag handle on SSR, with Next JS

### DISCUSSION THREAD

**Discussion by mtahagocer** [vercel/next.js#21922] [Help]
**What version of Next.js are you using?**

10.0.4

**What version of Node.js are you using?**

15.5.1

**What browser are you using?**

chrome

**What operating system are you using?**

macOS

**How are you deploying your application?**

vercel

**Describe the Bug**

On every click and every page refresh , output console
error

_document.js
import { ServerStyleSheets } from '@material-ui/core/styles'; import { resetServerContext } from 'react-beautiful-dnd';

```
  MyDocument.getInitialProps = async (ctx) => {
      const sheets = new ServerStyleSheets();
      const originalRenderPage = ctx.renderPage;
      ctx.renderPage = () => originalRenderPage({
        enhanceApp: (App) => (props) => {
          resetServerContext();
          return sheets.collect(<App {...props} />);
        },
      });
```

So, how I'm gonna fix it?

**Expected Behavior**

?

**To Reproduce**

?

![106482034-1abba100-64be-11eb-995a-84d432e7c390](https://user-images.githubusercontent.com/46215831/107121345-efacc500-68a2-11eb-9c3e-213152538af1.png)

**Comment by ifeora-emeka** [comment]
![image](https://user-images.githubusercontent.com/34845973/111895089-6abfda00-8a10-11eb-952c-ebb8db3d235a.png)

Having issues with react-beautiful-dnd as well

**Reply by ifeora-emeka** [reply]
@mtahagocer where you able to resolve this issue??

**Comment by ifeora-emeka** [comment]
I tried this and it worked for me -> https://codedaily.io/tutorials/185/Using-react-beautiful-dnd-with-NextJS

**Reply by mtahagocer** [reply]
Unfortunately this didn't work for me.
